.github/workflows/release.yml

```diff
@@ -31,11 +31,9 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - run: >
-          gh release create
-          --target ${{ github.event.pull_request.merge_commit_sha }}
-          {--title=CML\ ,}$(basename ${{ github.head_ref }})
-          --generate-notes
-          --draft
+          gh release create --target ${{
+          github.event.pull_request.merge_commit_sha }} {--title=CML\
+          ,}$(basename ${{ github.head_ref }}) --generate-notes --draft
```

Comment:
old formatting was better imo 👀

Reply:
I think this was auto-formatted by the linter.

Reply:
should not happen now with #1294 merged

Reply:
Not in the diff anymore 😅